tests/framework/cli/conftest.py

```diff
@@ -87,6 +87,7 @@ def fake_metadata(fake_root_dir):
         "CLI Testing Project",
         fake_root_dir / REPO_NAME,
         kedro_version,
+        kedro_version,
```

Comment:
`kedro_version` seems to be duplicated here now, is that necessary?

Reply:
As @idanov's comments recommend we should keep `project_version` in `ProjectMetadata` as it would be a breaking change otherwise. Since the `ProjectMetadata` class is a factory method, `NamedTuple`, there didn't seem to be a way to create a direct alias for `project_version` to `kedro_init_version`. Instead, I've kept both the `project_version` and `kedro_init_version` properties in `ProjectMetadata` as a temporary solution until 0.19. They both will always have the same value.

Not sure if this is the best approach?

As a result, when doing the unit tests we now have to take into account both `project_version` and `kedro_init_version` hence the repeated `kedro_version`.